yt/utilities/lib/interpolators.pyx

```diff
@@ -163,3 +163,47 @@ def ghost_zone_interpolate(int rf,
                 opos[2] += ods[2]
             opos[1] += ods[1]
         opos[0] += ods[0]
+
+@cython.cdivision(True)
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def fix_nonperiodic(np.ndarray[np.float64_t, ndim=3] field,
+                    np.ndarray[np.int_t,     ndim=1] lo_offset,
+                    np.ndarray[np.int_t,     ndim=1] hi_offset):
+    cdef int i, j, k
+    cdef int ii, ij, ik
+    cdef int nx, ny, nz
+    cdef int lx, ly, lz
+    cdef int hx, hy, hz
+    cdef np.float64_t xp, xm, yp, ym, zp, zm
+    cdef np.float64_t qp, qm
+
+    nx = field.shape[0]
+    ny = field.shape[1]
+    nz = field.shape[2]
+    lx = lo_offset[0]
+    ly = lo_offset[1]
+    lz = lo_offset[2]
+    hx = hi_offset[0]
+    hy = hi_offset[1]
+    hz = hi_offset[2]
```

Comment:
Assuming this is valid Cython code, I'd suggest for readability

```suggestion
    nx, ny, nz = field.shape
    lx, ly, lz = lo_offset
    hx, hy, hz = hi_offset
```

Reply:
looks like tuple unpacking is indeed supported in Cython, for instance we use it here: https://github.com/yt-project/yt/blob/8c8dd9bf2eb767c9bcfaef72a4d069a2067e6777/yt/utilities/lib/pixelization_routines.pyx#L1562

Reply:
Yep! Was thinking of changing this out as it seemed excessively large.